packages/jobs/rollup.config.mjs

```diff
@@ -0,0 +1,43 @@
+import * as path from 'path'
+import * as url from 'url'
+
+import alias from '@rollup/plugin-alias'
+import commonjs from '@rollup/plugin-commonjs'
+import typescript from '@rollup/plugin-typescript'
+import { dts } from 'rollup-plugin-dts'
+
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
+const aliasEntries = {
+  entries: [{ find: '$jobs', replacement: path.resolve(__dirname, 'src') }],
+}
+
+const COMMON_PLUGINS = [
+  typescript({
+    noEmit: true,
+    tsconfig: './tsconfig.json',
+    exclude: ['**/__tests__', '**/*.test.ts', '**/*.d.ts'],
+  }),
+  commonjs(),
+]
+const EXTERNAL = ['ioredis', 'bullmq']
+
+/** @type {import('rollup').RollupOptions[]} */
+export default [
+  {
+    input: 'src/index.ts',
+    output: [{ file: 'dist/index.js' }],
+    plugins: COMMON_PLUGINS,
+    external: EXTERNAL,
+  },
```

Comment:
I assume index.ts exports the code that is used in web and api? you don't really need to build it no? It can be exported as typescript to the rest of the projects? 

src/server.ts is the only thing that needs building in truth. Or am I missing something?

Reply:
Probably you're right. I'll cleanup when fixing docker build